### ISSUE TITLE
webpack aliased imports in scss file ignored

### ISSUE THREAD

**Issue by hoorncj** [depcheck/depcheck#528]
I have a monorepo project, with a file structure such as this:

```  
package.json
packages
   my-component
       my-component.scss
       my-component.jsx
       package.json
   styles
       _styles.scss
       package.json
```

packages build into a `/dist` folder, so `_styles.scss` end up at `node_modules/styles/dist/_styles.scss`

my-component.scss references _styles.scss via a webpack style alias ~
```css
@import '~styles/dist/_styles.scss';

.class {
    color: [use some variable from partial];
}
```

If I add 'styles' package to my-component's package.json and run depcheck, it tells me 'styles' is unused.

**Comment by yingzhe** [comment]
#536

**Comment by rumpl** [comment]
@hoorncj I just merged #536, the fix will be available in the next release (coming soon).
